cmake/modules/grpc.cmake

```diff
@@ -25,35 +25,46 @@ elseif(NOT USE_BUNDLED_GRPC)
 else()
 	include(cares)
 	include(protobuf)
+	include(openssl)
 	find_package(PkgConfig)
 	if(NOT PKG_CONFIG_FOUND)
 		message(FATAL_ERROR "pkg-config binary not found")
 	endif()
 	set(GRPC_SRC "${PROJECT_BINARY_DIR}/grpc-prefix/src/grpc")
 	set(GRPC_INCLUDE "${GRPC_SRC}/include")
-	set(GRPC_LIB "${GRPC_SRC}/libs/opt/libgrpc_unsecure.a")
-	set(GRPCPP_LIB "${GRPC_SRC}/libs/opt/libgrpc++_unsecure.a")
+	set(GRPC_LIB "${GRPC_SRC}/libs/opt/libgrpc.a")
+	set(GRPCPP_LIB "${GRPC_SRC}/libs/opt/libgrpc++.a")
 	set(GRPC_CPP_PLUGIN "${GRPC_SRC}/bins/opt/grpc_cpp_plugin")
 
 	get_filename_component(PROTOC_DIR ${PROTOC} PATH)
 
 	if(NOT TARGET grpc)
 		message(STATUS "Found pkg-config executable: ${PKG_CONFIG_EXECUTABLE}")
 		message(STATUS "Using bundled grpc in '${GRPC_SRC}'")
+
 		ExternalProject_Add(grpc
 			PREFIX "${PROJECT_BINARY_DIR}/grpc-prefix"
-			DEPENDS protobuf zlib c-ares
-			URL "http://download.draios.com/dependencies/grpc-1.8.1.tar.gz"
-			URL_MD5 "2fc42c182a0ed1b48ad77397f76bb3bc"
+			DEPENDS openssl protobuf c-ares
```

Comment:
What happened to zlib?

Reply:
See the [comment below](https://github.com/falcosecurity/libs/pull/30#discussion_r614708946).